### PR TITLE
Removed Julia 0.5 deprications

### PR DESCRIPTION
**Pull request by mauro3**

### CODE REVIEW

File: src/proj_capi.jl [hunk 1]
```diff
@@ -69,7 +69,7 @@ end
 
 "Get human readable error string from proj.4 error code"
 function _strerrno(code::Cint)
-    bytestring(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
+    @compat unsafe_string(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
```
Comment (c42f) [inline src/proj_capi.jl]: This is not a syntax change, so `@compat` should not be needed here - just `using Compat` should do the trick (From Compat's README it looks like we'll need at least Compat v0.8.0 in the REQUIRE to get `unsafe_string`).

Comment (mauro3) [inline src/proj_capi.jl]: Good point, fixed.

Comment (c42f) [inline src/proj_capi.jl]: Perfect, thanks.

File: .travis.yml [hunk 2]
```diff
@@ -3,8 +3,9 @@ os:
   - linux
   - osx
 julia:
-#  - release # 0.3 not supported yet
-  - nightly
+  - 0.4
+  - 0.5
+# - nightly
```
Comment (tkelman) [inline .travis.yml]: instead of commenting this out, make it an allowed failure so it'll run but won't make the status red?

Comment (tkelman) [inline .travis.yml]: https://docs.travis-ci.com/user/customizing-the-build#Rows-that-are-Allowed-to-Fail

Comment (mauro3) [inline .travis.yml]: Ok, for the travis-challenged like myself:
```
...
julia:
  - 0.4
  - 0.5
  - nightly
matrix:
  allow_failures:
    - julia: nightly
...
```

**Pull request merged**